docs/grid-limit-numeric-filter-value.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Limit value in a numeric filter
+description: An example on how to limit the value in a numeric filter.
+type: how-to
+page_title: Limit value in a numeric filter
```

Comment:
The page title has to include the main component we are using 

 Limit value in a numeric filter | KendoReact ComponentName